js/sdk/src/contexts/Hub/hub.js

```diff
@@ -1209,7 +1209,10 @@ const hubContextHelper = ({
         }
       })
       setHubState((prevState) => ({ ...prevState, ...updatedHubState }))
-      setHubContentState(updatedHubContent)
+      setHubContentState((prevState) => ({
+        ...prevState,
+        ...updatedHubContent,
```

Comment:
`updatedHubContent` should be initialized as empty obj on line 1195, believe this unintentionally bypasses the diffing done by prevState + spread operator